var/spack/repos/builtin/packages/py-quantities/package.py

```diff
@@ -11,20 +11,26 @@ class PyQuantities(PythonPackage):
 
     homepage = "https://python-quantities.readthedocs.org"
     pypi = "quantities/quantities-0.12.1.tar.gz"
-    maintainers("apdavison")
+    maintainers = ["apdavison"]
```

Comment:
Please restore the new directive (see https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers).